pallets/msa/src/lib.rs

```diff
@@ -646,33 +646,29 @@ pub mod pallet {
 
 		/// Retire a MSA
 		///
+		/// When a user wants to disassociate themselves from Frequency, they can retire their MSA for free provided that:
+		///  (1) They own the MSA
+		///  (2) There is only one account key
+		///  (3) The MSA is not a registered provider.
+		///
+		/// This does not currently remove any messages related to the MSA.
```

Comment:
Can we add a documentation for #Arguments that this extrinsic takes, you can check other examples of adding arguments like ``add_key_to_msa```

Reply:
Definitely.  I see many other functions that are missing documentation for #Arguments as well.  We should document all of them but I'll add them for this one function.

Reply:
So for the `retire_msa()` function, it just takes in the `origin` as an argument.  I will document it.